index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

Just a top level comment that per lab instructions, all your files should go into a  directory called ``lab-<your-name>``, i.e., all your lab work goes into a directory called ``lab-kaylee``.